client/src/components/edit/index.js

```diff
@@ -1,35 +1,81 @@
 import React, { Component } from 'react'
-import { Experience } from './Experience'
 import './edit.css'
-import { Personal } from './Personal'
-import { Media } from './Media'
+import Personal from '../../containers/personal'
+import Media from '../../containers/media'
 import { style } from '../../widgets/styles'
 import Avatar from 'material-ui/Avatar'
-import Person from 'material-ui/svg-icons/social/person'
+import {RaisedButton} from "material-ui";
+import Experiences from './Experiences'
+
+
+
+//function for the update of existing information
+export async function showResults(values) {
+    console.log(`You submitted:\n\n${JSON.stringify(values, null, 2)}`);
+}
+
+//function for the post of new experiences
+export async function printResults(values) {
+    console.log(`halala submitted:\n\n${JSON.stringify(values, null, 2)}`);
+}
+
+function clickFunction(submit){
+  //update
+    submit('personal');
+    submit('existingExperiences')
+    submit('media')
+
+    submit('newExperiences'); //post
+}
 
 class EditProfile extends Component {
+  componentDidMount() {
+    const { fetchProfessional } = this.props
+    fetchProfessional(3)
```

Comment:
I think this should be left out since fetchUser is called when the app initializes and when the user signs in. There are a couple other components that call fetchProfessional, resulting in 3 of the same API calls.